src/main/index.ts

```diff
@@ -35,6 +35,25 @@ protocol.registerSchemesAsPrivileged([
   {
     scheme: 'webxdc',
     privileges: {
+      // This gives apps access to APIs such as
+      // - Web Cryptography
+      // - Clipboard
```

Comment:
probably not, but I guess we should test against that before merging

Reply:
I have tested it. It's already the case, even without `isSecureContext`. Try

```javascript
window.addEventListener('DOMContentLoaded', () => {
  const textarea = document.createElement('textarea');
  document.body.appendChild(textarea);
  textarea.focus();
  setTimeout(() => {
    document.execCommand('paste');
  });
});
```

Reply:
So I removed the "Clipboard" point because yeah, it requires permission, which is currently denied (the `write()` API too)  (but `document.execCommand('paste')` still works, but it's not directly related to this MR).

Reply:
That is to say, clipboard-related security is not affected by this MR.